### ISSUE TITLE
No Default ImputMap actions for character controller

### ISSUE THREAD

**Issue by DeficientNull** [Phazorknight/Cogito#348]
**Cogito and Godot Engine Version:**

Latest for both

**Description:**
I expect to run the example scenes with existing Input Actions since the docs imply that. This doesn't happen though and the docs don't go into detail about how to set that up.

**Reproduction steps:**
Run the latest version of the project and attempt to walk or assign your own controls for movement.

**Comment by Phazorknight** [comment]
Hey, sorry about this.

Unfortunately, the way Godot stores the InputMap inside the project settings, there's no super-easy way to include them when grabbing Cogito from the AssetLibrary.

I recommend using the whole Cogito project from this GitHub repo. That way, the input map is preconfigured and you should be good to go. We'll also update the documentation to be clearer about this.

We were experimenting with adding a button that would "reset" the project input map configuration to provide this, but the implementation was a bit questionable, will look into improving this in future updates though.

**Comment by DeficientNull** [comment]
Right, yes i eventually found that out. One temporary fix is to just have the input map names in the documentation. Simply to make sure people know how to set them up manually if they didn't realize they needed project.godot like me.

Regardless, thank you for the speedy reply. Loving the template so far.

On another note, it seems part of the weapon material is broken and does not display properly. This time i'm not sure it's an issue with missing files. Something to do with a global shader parameter `viewmodel_fov` being missing.

**Comment by Phazorknight** [comment]
@DeficientNull 
Yeah, I'll update the documentation soon to make this clearer and include a list of input map actions, now that I'm back from my winter break.

Re weapon material: Would you be able to open a new issue with this with a few more details? (which mesh/material, how it's broken?)

**Comment by Phazorknight** [comment]
@DeficientNull 
I've added a function to reset your projects input map settings (which will also add any needed input map settings on a blank project) in this commit: https://github.com/Phazorknight/Cogito/commit/0fd1fa327c1ce5e3970548a2350f3fac222a26ef

Give it a spin when you have a second, should help getting projects off the ground more easily.
